bvm/ballerina-runtime/README.md

```diff
@@ -0,0 +1,256 @@
+# The Ballerina Java Runtime
+This module contains the implementation of the Ballerina runtime which is designed as the following packages.
+
+#### io.ballerina.runtime.internal.\*
+Since the internal package is restricted only to be used with Ballerina runtime internals, ballerina developers **should not** use any java classes from the internal package. These internal java constructs can be often changed and will not be part of the release note.
```

Comment:
```suggestion
Since the internal package is restricted only to be used with Ballerina runtime internals, Ballerina developers **should not** use any java classes from the internal package. These internal java constructs can be often changed and will not be part of the release note.
```
Shall we change `ballerina` to `Ballerina` in the texts. There are several text occurrences.